lib/tapyrus/rpc/tapyrus_core_client.rb

```diff
@@ -62,6 +89,7 @@ def request(command, *params)
         request.content_type = 'application/json'
         request.body = data.to_json
         response = http.request(request)
+        raise ConnectionError.new(response) unless response.is_a? Net::HTTPOK
```

Comment:
If you raise an exception here, you do not need below code, isn't it?

```
raise response['error'].to_json if response['error']
```

Reply:
That code is need. Because tapyrus rpc server sometimes response error message with HTTP 200 OK. For example argument error case.

My adding code only raise when http status is not 200.

Maybe, Should I wrap in this error message to same Error class?

Reply:
> For example argument error case.

Can you write those actual code?

Reply:
Sorry, I was missunderstanding.

RPC-Server return 500 status code when anything problem in args.
```
< HTTP/1.1 500 Internal Server Error
< Content-Type: application/json
< Date: Mon, 01 Mar 2021 08:31:26 GMT
< Content-Length: 148
< 
{"result":null,"error":{"code":-19,"message":"Wallet file not specified (must request wallet RPC through /wallet/<filename> uri-path)."},"id":null}
```

I will update that code.